tests/cnf/ran/oran/tests/oran-provision.go

```
@@ -183,9 +193,13 @@ func verifyProvisioningRequestFulfilled(prBuilder *oran.ProvisioningRequestBuild
 	By("verifying the InfrastructureResourceStatuses are set")
 
 	resourceStatuses := status.Extensions.InfrastructureResourceStatuses
-	if len(resourceStatuses) == 0 {
-		accumulatedErrors = append(accumulatedErrors,
-			fmt.Errorf("expected provisioned infrastructure nodes in extensions"))
+	expectedHostnames, err := helper.GetSpokeHostnames()
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (too many statements above if)

(wsl_v5)